Utils/github_workflow_scripts/handle_external_pr.py

```diff
@@ -180,14 +200,60 @@ def is_requires_security_reviewer(pr_files: list[str]) -> bool:
     return False
 
 
+def is_tim_content(pr_files: list[str]) -> bool:
+    """
+    This is where the actual search for feed:True or relevant tags or categories are being searched
+    according to the login in is_tim_reviewer_needed
+
+    Arguments:
+    - pr_files: List[str] The list of files changed in the Pull Request.
+
+    Returns: returns True or False if tim reviewer needed
+    """
+    integrations_checked = []
+    for file in pr_files:
+        integration = BaseContent.from_path(CONTENT_PATH / file)
```

Comment:
This returns the same value for both panos.py and panos.yml?

Reply:
yes - the integration name will be returned